PSix/FBRequest.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  FBRequest.swift
+//  PSix
+//
+//  Created by Turzion, Avihu on 4/20/15.
+//  Copyright (c) 2015 PayPal. All rights reserved.
+//
+
+import Foundation
+import Parse
+
+
+typealias FBParamsList = [NSObject: AnyObject]
+
+let URI_SEP = "/"
+
+
+class FBRequest {
+    
+    private(set) var path: String
+    private(set) internal var params: FBParamsList
+    
+    init(forResource: String, withParams: FBParamsList = FBParamsList()) {
+        path = forResource
+        params = withParams
+    }
+    
+    func addParam(param: String, value: AnyObject) -> FBRequest {
+        params[param] = value
+        return self
+    }
+    
+    func edge(name: String) -> FBRequest {
+        return FBRequest(forResource: path + URI_SEP + name)
+    }
+    
+    func execute(failure: ((NSError) -> ())? = nil, success: (FBResponse) -> ()) {
+        if FBSDKAccessToken.currentAccessToken() != nil {
```

Comment:
I'm thinking about removing this test altogether as this should only wrap the graph request and as with the regular Facebook APIs you, the SDK user, need to check for a valid token before issuing a request.